Src/LinearSolvers/MLMG/AMReX_MLMG.cpp

```diff
@@ -633,14 +639,15 @@ MLMG::interpCorrection (int alev)
             {
                 const Box& fbx = mfi.tilebox();
                 const Box& cbx = amrex::coarsen(fbx,2);
-                const Box& tmpbx = amrex::refine(cbx,2);
+                Box tmpbx = amrex::refine(cbx,2);
+		if (nghost > 1) tmpbx.grow(2);
                 tmpfab.resize(tmpbx,ncomp);
                 amrex_mlmg_lin_nd_interp(BL_TO_FORTRAN_BOX(cbx),
                                          BL_TO_FORTRAN_BOX(tmpbx),
                                          BL_TO_FORTRAN_ANYD(tmpfab),
                                          BL_TO_FORTRAN_ANYD(cfine[mfi]),
 					 &ncomp);
-                fine_cor[mfi].copy(tmpfab, fbx, 0, fbx, 0, ncomp);
+                fine_cor[mfi].copy(tmpfab, tmpbx, 0, tmpbx, 0, ncomp);
```

Comment:
Note that even when nghost is zero, tmpbx may be different from fbx.

Reply:
```suggestion
fine_cor[mfi].copy(tmpfab, tmpbx, 0, tmpbx, 0, ncomp);
```
This is what we currently have, so I may not understand the suggestion. Using `fbx` instead of `tmpbox` definitely breaks the code; the correction must be copied to the ghost nodes. What about
```suggestion
Box tmpbx2 = fbx;
if (nghost>1) tmpbx2.grow(nghost);
...
fine_cor[mfi].copy(tmpfab, tmpbx2, 0, tmpbx2, 0, ncomp);
```

Reply:
That works.